content/3.workflow/2.BNO-onboarding.md

```diff
@@ -4,6 +4,14 @@ The BNO onboarding process consists of the following steps:
 
 1. The PSP provides HTTPS endpoint details, and assigns a Network ID to the BNO.
 2. The BNO provides HTTPS endpoint details to the PSP.
+
+::notice{info}
+HTTPS endpoint details designate a REST API configuration where:
+
+1. the payment initiation requests are sent to
+2. status updates are reported to
+::
+
```

Comment:
This notice contains the same information as in the list but in other words. I guess these details should be added right into the list.